packages/common/integration/tests/directives/filter-characters/filter-characters-form.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ChangeDetectionStrategy, Component, DebugElement } from '@angular/core';
+import { FormBuilder, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { FilterCharactersModule } from '@angular-ru/common/directives';
+import { NgxMaskModule } from 'ngx-mask';
+import { MatInputModule } from '@angular/material/input';
+import { BrowserModule, By } from '@angular/platform-browser';
+
+describe('[TEST]: FilterCharacters Input', () => {
+    let fixture: ComponentFixture<TestComponent> | null = null;
+    let component: TestComponent | null = null;
+    let debugElement: DebugElement | null = null;
+
+    @Component({
+        selector: 'test',
+        template: `
+            <div [formGroup]="form">
```

Comment:
`[filterCharacters]` instead `filterCharacters`?

Reply:
funny mistake

Reply:
interesting, strange that it worked